### PR TITLE
Remove spec_url from api.Document.execCommand

### PR DESCRIPTION
**Pull request by SamB**
The only mentions of execCommand I saw in Trusted Types were in the names of tests.

According to <https://respec.org/xref/?term=execCommand%28%29>, this API is defined in just one spec, so use that one.

### CODE REVIEW

**Comment by SamB** [comment]
It is true that the spec doesn't exactly have good standing, but it is the only one we appear to have, and the api is implemented ~everywhere.

**Comment by queengooborg** [comment]
It looks like the spec URL you had tried to use is an unofficial draft, which Webref does not consider as a valid standard.  Upon further research, it appears that the feature was originally defined in the [First Public Working Draft of the trusted types spec](https://www.w3.org/TR/2022/WD-trusted-types-20220927/#integration-with-exec-command), but was then removed in the [following Working Draft](https://www.w3.org/TR/2024/WD-trusted-types-20240531/).

As such, I think that it's actually better to mark this as non-standard and remove the spec URL.

**Comment by SamB** [comment]
Note that html appears to make quite a few normative references to execCommand. The [Editing APIs](https://html.spec.whatwg.org/#editing-apis) section, in particular, appears to incorporate by reference all of the following:

    {
      "https://w3c.github.io/editing/docs/execCommand/": {
        "anchors": [
          "execcommand%28%29",
          "querycommandenabled%28%29",
          "querycommandindeterm%28%29",
          "querycommandstate%28%29",
          "querycommandsupported%28%29",
          "querycommandvalue%28%29",
          "editable",
          "active-range",
          "editing-host-of",
          "delete-the-selection"
        ]
      }
    }

(that's every anchor it references from execComman, as listed at <https://github.com/w3c/webref/blob/48fcf5576be961ce5bc6cd2a134301b572c5952d/ed/links/html.json#L2807>) as well as some user interface requirements. (It also apparently attempts to include a definition of text selection, which presumably should now refer to <https://w3c.github.io/selection-api/#dfn-selection> instead.)

File: api/Document.json [hunk 1]
```diff
@@ -2941,7 +2941,7 @@
       "execCommand": {
         "__compat": {
           "mdn_url": "https://developer.mozilla.org/docs/Web/API/Document/execCommand",
-          "spec_url": "https://w3c.github.io/trusted-types/dist/spec/#dom-document-execcommand",
+          "spec_url": "https://w3c.github.io/editing/docs/execCommand/#dom-document-execcommand",
```
[the remark this answers is not in the document]
Comment (caugner) [reply api/Document.json:2944]: /cc @Elchi3 for a second opinion.

[the remark this answers is not in the document]
Comment (Elchi3) [reply api/Document.json:2944]: @tidoust do you know if there is an update on the state of the execCommand spec? In summer we marked it as "pending" in https://github.com/w3c/browser-specs/issues/1429. Maybe @zcorpan can say more, too?

[the remark this answers is not in the document]
Comment (tidoust) [reply api/Document.json:2944]: No update that I'm aware of. The spec has not been updated and still starts with the note that says that it's been implemented everywhere but inconsistently. The [README](https://github.com/w3c/editing/?tab=readme-ov-file#potential-deliveries-2) still says the spec is a "potential deliverable" of the Web Editing Working Group.

**Comment by caugner** [comment]
@Elchi3 While waiting for the spec state change, should we either keep the (broken) spec link as is (i.e. close this PR), or remove the `spec_url`? I would prefer to not keep this PR hanging around.

File: api/Document.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (zcorpan) [reply api/Document.json:2944]: I haven't started to work on the spec yet, but it's on my list to do. You could either drop the spec link for now and add it back in when the spec is better maintained, or keep the spec link.

**Review by caugner — APPROVED** [review]

**Comment by jpc-ae** [comment]
[w3c lists the spec url](https://github.com/w3c/browser-specs/blob/c76964c80f476c18a235847b2ce55be8d68e9939/specs.json#L514C13-L514C60) as https://w3c.github.io/editing/docs/execCommand/

Considering the ubiquitousness of this command, could the spec_url not be updated to point there rather than removing it completely? I get that it's an "unofficial draft", but until it starts being taking out of browsers, I don't see a reason not to include it.

**Comment by caugner** [comment]
@jpc-ae See the discussion above. As per our guidelines, we don't link to "unofficial" specs.

**Pull request merged**